lua/lspconfig/server_configurations/omnisharp.lua

```diff
@@ -39,7 +39,13 @@ return {
 
     filetypes = { 'cs', 'vb' },
     root_dir = function(fname)
-      return util.root_pattern '*.sln'(fname) or util.root_pattern '*.csproj'(fname)
+      local root_patterns = { '*.sln', '*.csproj', 'omnisharp.json', 'function.json' }
+      for _, pattern in ipairs(root_patterns) do
```

Comment:
no need do for range in there just

```lua
return util.root_pattern(patterns)(fname),
```

Reply:
Thanks for the feedback, I let copilot write the code :)